src/encoded/types/cart.py

```diff
@@ -96,3 +101,51 @@ def get_or_create_cart_by_user(context, request):
         '@type': ['result'],
         '@graph': carts or [cart]
     }
+
+
+@view_config(context=Cart.Collection, request_method='PUT', permission='save-carts', name='put-cart')
+def create_cart_by_user(context, request):
+    userid = get_userid(request)
+    user = _get_user(request, userid)
+    blocked_statuses = ['deleted'] if 'group.admin' not in request.effective_principals else []
+    carts = get_cart_objects_by_user(request, userid, blocked_statuses)
```

Comment:
Blocked_statuses also needs to be passed as keyword argument as above.

Reply:
OK